internal/bundle/basic_test.go

```diff
@@ -20,7 +20,7 @@ func TestAccBasicBundleDeployWithFailOnActiveRuns(t *testing.T) {
 	root, err := initTestTemplate(t, ctx, "basic", map[string]any{
 		"unique_id":     uniqueId,
 		"node_type_id":  nodeTypeId,
-		"spark_version": "13.2.x-snapshot-scala2.12",
+		"spark_version": "13.3.x-snapshot-scala2.12",
```

Comment:
Nitpick: maybe we can define a constant instead and just reuse it everywhere in the tests?

Reply:
Done.